src/detect-transform-pcrexform.c

```diff
@@ -31,9 +31,11 @@
 #include "detect-transform-pcrexform.h"
 #include "detect-pcre.h"
 
+#define PCRE_SERIALIZED_DATA_LEN    32
 typedef struct DetectTransformPcrexformData {
     pcre2_code *regex;
     pcre2_match_context *context;
+    uint8_t serialized_data[PCRE_SERIALIZED_DATA_LEN];
```

Comment:
could just use a ptr and strdup the input to the pcrexform keyword?

Reply:
I considered that - the regex string needs to be available during the `Serialize` call which complicates things a bit.